sklearn/linear_model/_ridge.py

```diff
@@ -767,7 +767,13 @@ def fit(self, X, y, sample_weight=None):
 class RidgeClassifier(LinearClassifierMixin, _BaseRidge):
     """Classifier using Ridge regression.
 
-    Read more in the :ref:`User Guide <ridge_regression>`.
+    This classifier first converts binary targets to ``{-1, 1}`` and then
+    treats the problem as a regression task with a
+    :class:`~sklearn.linear_model.Ridge` regressor. The predicted class
+    corresponds to the sign of the regressor's prediction.
```

Comment:
I can't understand this part, the decision function is related to the output of the regressors and the predicted class is related to the argmax of the decision function?

Reply:
Yes exactly.

Reply:
actually when n_classes=2, you're right :)